src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRanges.kt

```diff
@@ -23,6 +23,7 @@ import java.io.File
 /**
  * Data class to hold the information needed to output a BedRecord.
  */
+// TODO when we generate these objects, need to go from 1-based to 0-based indexing
```

Comment:
Should this say "TODO"?

Reply:
It's a note to myself that I forgot about, I'll remove it. Thanks for the catch!